README.md

```diff
@@ -108,6 +108,9 @@ vars:
     using_credit_card_payment_txn: true #enable if you want to include credit card payment transactions in your staging models
 ```
 
+### Customize the ordinal for account classes in balance sheet and profit-and-loss models
+If you're also using the `dbt_quickbooks` package, you can customize the order of your account classes in the `quickbooks__balance_sheet_account_class_order` and `quickbooks__profit_and_loss_account_class_order` seed files by changing the `ordinal` numbering. The current default numbering is based on best practices for finance balance sheets and profit-and-loss statements. If you are adding additional account class types, you will need to customize your own version of the `int_quickbooks__account_classifications` model to handle the logic for these types and either override or disable the above model and set up your own custom configuration. 
```

Comment:
Based on my suggestion in the below snippet, this will likely need to be updated.

Reply:
Updated!